### ISSUE TITLE
Where is your code?

### ISSUE THREAD

**Issue by Dinxin** [System-T/TextSimplification#1]

**Comment by Raidus** [comment]
Yeah. In the paper its stated "Our datasetsa nd code will be available at https:// github.com/System-T/TextSimplification :D

**Comment by yunyaoli** [comment]
Dataset is available: https://github.com/System-T/TextSimplification

For code, I believe that we got stuck with legal at some point. @huaiyu-zhu Do you know what's the status?
